examples/hwapi/hwconfig_pyboard.py

```diff
@@ -0,0 +1,4 @@
+from machine import Pin, Signal
+
+# pyboard has blue LED on pin 'B4', active high (not inverted)
+LED = Signal(Pin('B4', Pin.OUT))
```

Comment:
Better to use the named pin "LED_BLUE" because that's guaranteed to give you the blue LED on all current and future pyboards.

There are also 3 other LEDs and 1 button, so could add those following hwconfig_dragonboard410c.py